ssjetpackcomposeprogressbutton/src/main/java/com/simform/ssjetpackcomposeprogressbuttonlibrary/SSJetPackComposeProgressButton.kt

```diff
@@ -242,7 +260,10 @@ fun SSJetPackComposeProgressButton(
             enabled = enabled && buttonState != SSButtonState.LOADING,
             elevation = elevation,
             shape = RoundedCornerShape(ssAnimateIntAsState(cornerRadiusValue, speedMillis)),
-            border = buttonBorderStroke,
+            border = BorderStroke(
+                width = buttonBorderWidth ?: 0.dp,
```

Comment:
Replace with ZERO.dp

Reply:
We will be removing ZERO.dp in the upcoming PR.